libs/blocks/merch-card/merch-card.js

```diff
@@ -702,7 +711,7 @@ export default async function init(el) {
     merchCard.appendChild(footer);
 
     if (MULTI_OFFER_CARDS.includes(cardType)) {
-      const quantitySelect = extractQuantitySelect(el);
+      const quantitySelect = extractQuantitySelect(el, merchCard);
```

Comment:
in case there is an unexpected exception, I think it should log a lana error and silently ignore.